service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/content/C2UploadedEmailContentProvider.java

```diff
@@ -31,6 +33,19 @@
             .build();
     }
 
+    public Map<String, Object> buildC2UploadNotificationForAllocatedJudge(final CaseDetails caseDetails) {
+        CaseData caseData = mapper.convertValue(caseDetails.getData(), CaseData.class);
+
+        return ImmutableMap.<String, Object>builder()
+            .putAll(buildCommonNotificationParameters(caseDetails))
+            .put("callout", emailNotificationHelper.buildSubjectLineWithHearingBookingDateSuffix(caseData,
+                caseData.getHearingDetails()))
+            .put("respondentLastName", getFirstRespondentLastName(caseData.getRespondents1()))
+            .put("judgeTitle", caseData.getAllocatedJudge().getJudgeOrMagistrateTitle())
+            .put("judgeName", caseData.getAllocatedJudge().getJudgeName())
+            .build();
+    }
```

Comment:
I'd rather see this return a java object rather than a map. See AllocatedJudgeContentProvider for example.